### ISSUE TITLE
Regression(74e5fff77a9fbf): `pdf` can no longer read built-in fonts

### ISSUE THREAD

**Issue by nico** [SerenityOS/serenity#21856]
1. Get `pdffiles` here: https://github.com/nico/pdffiles
2. `Meta/serenity.sh build lagom pdf`
3. `Build/lagom/bin/pdf --debugging-stats ~/src/pdffiles/2ThestructureoftheCIE1997ColourAppearanceModelCIECAM97s.pdf`

Expected:

```
page number 10 / 10
101 times: Rendering of feature not supported: Type0 font CIDFontType2 not implemented yet
    on pages: 1 (101x)
1 times: Rendering of feature not supported: Image's bit per component != 8
    on pages: 3
```

Actual:

```
page number 10 / 10
752 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 2.5357845pt
    on pages: 2 (7x) 3 (37x) 4 (40x) 5 (21x) 6 (182x) 7 (168x) 8 (153x) 9 (144x)
362 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.5357845pt
    on pages: 2 (7x) 3 (23x) 4 (19x) 5 (7x) 6 (81x) 7 (112x) 8 (68x) 9 (45x)
349 times: Rendering of feature not supported: Can't draw text because an invalid font was in use
    on pages: 3 (37x) 4 (117x) 6 (4x) 7 (17x) 8 (9x) 9 (23x) 10 (142x)
307 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 1.5848039pt
    on pages: 3 (13x) 4 (19x) 5 (15x) 6 (81x) 7 (44x) 8 (61x) 9 (74x)
248 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.4382353pt
    on pages: 2 3 (2x) 4 (3x) 5 (15x) 6 (67x) 7 (46x) 8 (56x) 9 (57x) 10
186 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 2.4382353pt
    on pages: 4 (9x) 5 (4x) 6 (55x) 7 (46x) 8 (17x) 9 (55x)
101 times: Rendering of feature not supported: Type0 font CIDFontType2 not implemented yet
    on pages: 1 (101x)
51 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.0237746pt
    on pages: 10 (51x)
36 times: Internal error while processing PDF file: Failed to load Liberation Serif Bold at 2.0237746pt
    on pages: 10 (36x)
34 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 2.0237746pt
    on pages: 2 10 (33x)
30 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 1.5848039pt
    on pages: 5 (2x) 6 (7x) 7 (5x) 8 (15x) 9
22 times: Internal error while processing PDF file: Failed to load Liberation Serif Bold at 2.5357845pt
    on pages: 2 (2x) 3 (2x) 4 (5x) 5 (4x) 7 (2x) 8 (3x) 9 (4x)
20 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.2919118pt
    on pages: 2 (3x) 3 (2x) 4 (6x) 5 (4x) 7 (2x) 8 9 (2x)
12 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 0.24509805pt
    on pages: 3 4 5 (2x) 6 (3x) 8 (4x) 9
6 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 0.24509805pt
    on pages: 6 (2x) 7 8 (3x)
4 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.1943629pt
    on pages: 3 (4x)
4 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 1.5117649pt
    on pages: 5 (2x) 8 (2x)
3 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 9.972304pt
    on pages: 6 (2x) 8
3 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 9.972304pt
    on pages: 6 8 (2x)
3 times: Internal error while processing PDF file: Failed to load Liberation Serif Italic at 2.2919118pt
    on pages: 8 9 (2x)
2 times: Internal error while processing PDF file: Failed to load Liberation Serif Bold at 1.5848039pt
    on pages: 2 4
2 times: Internal error while processing PDF file: Failed to load Liberation Serif Bold at 0.24509805pt
    on pages: 5 10
2 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 1.9504904pt
    on pages: 2 (2x)
1 times: Internal error while processing PDF file: Failed to load Liberation Sans Regular at 0.9803922pt
    on pages: 10
1 times: Rendering of feature not supported: Image's bit per component != 8
    on pages: 3
1 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 2.5112746pt
    on pages: 4
1 times: Internal error while processing PDF file: Failed to load Liberation Serif Regular at 4.3887258pt
    on pages: 2
```

According to `git bisect`, this started with 74e5fff77a9fbf.

@AtkinsSJ

**Comment by nico** [comment]
Oh, apparently also fixed by #21849.

**Comment by nico** [comment]
Fix got merged.
